GoogleCloudPlatform/microservices-demo
# Turn off GCP integrations by default

### Background 

Turns off Google Cloud Monitoring, Tracing, Profiler, and Debugger for all services by default, to re-enable out of the box support for local clusters. 

### Fixes 

#663  

### Change Summary

1) Comments out `DISABLE_PROFILER=1` etc for all services, thus turning off GCP integrations by default 
2) Payments service, for some reason, didn't look at these env variables so none of the GCP integraitons were toggle-able. fixed that. 

### Testing Procedure

1. Create a local cluster (kind or minikube)
2. Have a project ID ready to go, `PROJECT_ID` env field set (for GCR)
3. `skaffold run --default-repo=gcr.io/$PROJECT_ID/onlineboutique` this will test the `kubernetes-manifests`, *not* `release/` on a local cluster. that's what you want. 
4. Verify that all pods start up without errors.
